### ISSUE TITLE
this.on is not a function

### ISSUE THREAD

**Issue by nerdenough** [apb2006/basex-node#20]
Get this stacktrace when I run the server with the `basex.Session(host, port, user, pass)` line.

```
C:\Users\Brendan\workspace\node\letterdb\node_modules\basex\index.js:220
    this.on("drain", function() {
         ^

TypeError: this.on is not a function
    at Object.Session (C:\Users\Brendan\workspace\node\letterdb\node_modules\basex\index.js:220:10)
    at Object.<anonymous> (C:\Users\Brendan\workspace\node\letterdb\app\server.js:31:22)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Function.Module.runMain (module.js:447:10)
    at startup (node.js:141:18)
    at node.js:933:3
```

Node: 5.7.1 (latest)
npm: 3.6.0

**Comment by apb2006** [comment]
Hi,
Your report looks very similar to #17.
Can you run the examples and the tests successfully?
If yes please provide a complete code sample of the problem.
Regards /andy

**Comment by nerdenough** [comment]
Hmm, I tried the tests and they all pass except for the stress test sending 1000000 items. Additionally the examples that I tried all worked fine as well.

I have managed to fix the issue I was having though by stopping BaseX, reinstalling Node.js and your BaseX package and then starting everything up again. Not entirely sure what the issue was as my code is still the exact same as it was:

```javascript
var basex = require('basex');
var basexSession = new basex.Session('localhost', 1984, 'admin', 'admin'); // error was being thrown here
```

Perhaps something had messed up in my Node install. Anyway thanks for the help and sorry for the trouble. This is a really great package!
